client/common/directives/ModelField/ModelField.js

```diff
@@ -316,7 +316,18 @@ angular.module('dashboard.directives.ModelField', [
           if (!property.display.options) property.display.options = {};
           if (!property.display.options.format) property.display.options.format = "YYYY-MM-DD  h:mm A";
         }
-        
+
+        if (property.display.convertToLocalTime === false) {
```

Comment:
Can you add you combine 320 and 322 to do `if (scope.data[scope.key] && property.display.convertToLocalTime === false)`